src/xrGame/ai/monsters/state.cpp

```diff
@@ -1,4 +1,17 @@
 #include "StdAfx.h"
+
+#ifdef DEBUG
+namespace debug
+{
+class text_tree;
+}
+
+class CBaseMonster
```

Comment:
Why only for debug mode?

Reply:
Because problem was only in debug mode ( here is travis build status before this fix: https://travis-ci.org/q4a/xray-16/builds/463837730 ).
And it's based on "base_monster.h". It's only for debug mode in base_monster.h too:
https://github.com/OpenXRay/xray-16/blob/686f5db0b70cdbfb9f28786fc7d4e0e145b53085/src/xrGame/ai/monsters/basemonster/base_monster.h#L45